app/src/main/java/org/mozilla/focus/observer/LoadTimeObserver.kt

```diff
@@ -0,0 +1,58 @@
+package org.mozilla.focus.observer
+
+import android.os.SystemClock
+import android.support.v4.app.Fragment
+import android.util.Log
+import org.mozilla.focus.architecture.NonNullObserver
+import org.mozilla.focus.session.Session
+import org.mozilla.focus.telemetry.TelemetryWrapper
+import org.mozilla.focus.utils.UrlUtils
+
+object LoadTimeObserver {
```

Comment:
I'm not entirely sure the change to a static observer object is a good one. You're now passing in the fragment to the static method, which could leave a reference around to leak memory. I don't think it'll leak for very long as written though.